src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
+const App = () => {
+  const figures = [{ val: 'AC' }, { val: '+/-' }, { val: '%' }, { val: '/' }, { val: 7 }, { val: 8 }, { val: 9 }, { val: 'x' }, { val: 4 }, { val: 5 }, { val: 6 }, { val: '-' }, { val: 1 }, { val: 2 }, { val: 3 }, { val: '+' }, { val: 0 }, { val: '.' }, { val: '=' }];
+  const id = 'item';
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <div className="display">0</div>
+      {figures.map((fig) => (
+        <button className="press" type="button" key="fig.val" id={id + figures.indexOf(fig)}>{fig.val}</button>
+      ))}
```

Comment:
- [ ] Good job working on this but there are a few things that need to be taken care of. They are listed below
  - There should be a `Calculator` component inside the `components` directory. It should be class-based and imported into the `App` component.
  - The `App` component should be class-based.

Reply:
Thanks, I will make the changes.